components/image-combo/image-combo.less

```diff
@@ -30,7 +30,7 @@
 		margin-bottom: 0;
 	}
 
-	.text-block {
+	.colored-block {
```

Comment:
This class is not used anywhere, `<ImageCombo>` is used in the team overview component with no props and in the case page with the prop `classes={hasTextCard ? 'image-combo-text' : ''}`.

So the class can be deleted. And if you have the time I would remove the `classes` prop and replace it with something like `hasText`? Having a generic `classes` prop seems like a bad idea.

Reply:
On the case page `<QuoteBlock>` is a child of `<ImageCombo>`. `<QuoteBlock>` uses this class so we can't delete it.

Reply:
I did add a new `hasText` prop and removed `classes`.

Reply:
Okay, not sure if this is technical debt or not...